portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,72 @@
+package com.google.sps.data;
+
+public class Comment{
```

Comment:
If you had read about making every entity Immutable (from Effective Java), then Builder pattern makes sense here.

Reply:
Yes, Will do.